# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openglad/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur openglad-0.98/src/parser.cpp openglad-0.98.new/src/parser.cpp
--- openglad-0.98/src/parser.cpp       2007-12-10 02:26:31.000000000 -0500
+++ openglad-0.98.new/src/parser.cpp   2007-12-10 02:27:17.000000000 -0500
@@ -23,6 +23,8 @@
 #include <iostream>
 #include <fstream>
 #include <string>
+#include <cstring>
+#include <cstdlib>
 #include "parser.h"
 
 using namespace std;

